.github/workflows/docker-build-and-push-main.yaml

```diff
@@ -70,18 +72,38 @@ jobs:
           mkdir src
           vcs import src < autoware.repos
 
-      - name: Build 'autoware-openadk'
+      - name: Build 'Autoware'
         uses: ./.github/actions/docker-build-and-push
         with:
-          bake-target: autoware-openadk
+          bake-target: autoware
           build-args: |
             *.platform=linux/amd64
             *.args.ROS_DISTRO=${{ needs.load-env.outputs.rosdistro }}
             *.args.BASE_IMAGE=${{ needs.load-env.outputs[format('{0}', matrix.base_image_env)] }}
             *.args.SETUP_ARGS=${{ matrix.setup-args }}
             *.args.LIB_DIR=${{ matrix.lib_dir }}
-          tag-suffix: ${{ matrix.additional-tag-suffix }}-amd64
+          tag-suffix: ${{ matrix.additional-tag-suffix }}
+          tag-arch: -amd64
```

Comment:
Are there any reasons for splitting `-amd64` as separate argument as  `tag-arch`?
(I know it would be cleaner, but I was curious if you needed to make the change)

Reply:
The thing is in the near future we would use hybrid build platform(AWS or buildjet) both for building arm and x86 architecture then it would be easier to define architecture prior to going into actual github action.